src/components/Player/Player.component.jsx

```diff
@@ -0,0 +1,135 @@
+
+const PlayerContainer = styled.div`
+  height: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-top: 3px;
+  max-width: 1200px;
+  min-width: 1200px;
+`;
+
+const TextDescription = styled.p`
+  color: ${(props) => props.theme.textcolor};
+`;
+
+const ActionsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap-reverse;
+  justify-content:initial;
+  align-items:center:
+`;
+
+const TextDescriptionsContainer = styled.div`
+  width: 70%;
+`;
+
+const FadeOut = keyframes`
+  from {
+    transform: translateX(0px);
+    opacity: 1;
+  }
+
+  to {
+    transform: translateY(-250px);
+    opacity: 0;
+  }
+`;
+
+const ButtonsContainer = styled.div`
+  margin-left: 30px;
+`;
+
+const Emoji = styled.h1`
+  position: relative;
+  bottom: 0;
+  font-size: 125px;
+  transform-origin: center;
+  animation: ${FadeOut} 1.5s ease infinite;
+`;
+
+const FavoritesButton = styled.button`
+  margin-top: 10px;
+  background: ${(props) => props.theme.primary};
+  color: white;
+  padding: 10px;
+  border-radius: 8px;
+`;
+
+function Player(props) {
+  const { setCurrentVideo, videos, currentVideo } = useContext(VideoContext);
+  const {
+    removeFromFavoritesProvider,
+    addToFavoritesProvider,
+    isFavoriteVideoProvider,
+  } = useContext(FavoritesContext);
+  const { authenticated } = useAuth();
+  const [isFavorite, setIsFavorite] = useState(false);
+  const [showEmoji, setShowEmoji] = useState(false);
+
+  useEffect(() => {
+    if (videos !== undefined) {
+      const videoId = props.id;
+      const video = videos.find((actualVideo) => actualVideo.videoId === videoId);
+      if (video !== undefined) {
+        setCurrentVideo(video);
+        setIsFavorite(isFavoriteVideoProvider(video));
+      }
+    }
+  }, [videos, setCurrentVideo, props.id, isFavoriteVideoProvider]);
+
+  const addToFavoritesHandleClick = () => {
+    setShowEmoji(true);
+    setTimeout(() => setShowEmoji(false), 1000);
+    addToFavoritesProvider(currentVideo);
+    setIsFavorite(true);
+  };
+
+  const removeFromFavoritesHandleClick = () => {
+    setShowEmoji(true);
+    setTimeout(() => setShowEmoji(false), 1000);
+    removeFromFavoritesProvider(currentVideo);
+    setIsFavorite(false);
+  };
+
+  const url = `https://www.youtube.com/embed/${props.id}`;
+  return (
+    <PlayerContainer>
+      <iframe
+        title={props.id}
+        width="100%"
+        height="500"
+        src={url}
+        frameBorder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowFullScreen
+      />
+      <ActionsContainer>
+        <TextDescriptionsContainer>
+          <TextDescription>{currentVideo.title}</TextDescription>
+          <TextDescription>{currentVideo.description}</TextDescription>
+        </TextDescriptionsContainer>
+        <ButtonsContainer hidden={!authenticated}>
+          <Emoji hidden={!showEmoji} offset={200}>
```

Comment:
You might want to change the position to absolute and add `position: relative` to the ButtonsContainer, so the animation would not move the button down
<img width="217" alt="Screen Shot 2020-12-16 at 20 47 17" src="https://user-images.githubusercontent.com/6569533/102442855-9af77700-3fe2-11eb-8796-385d3a8fcf5f.png">

Reply:
Actually it should be hidden  while showing the emoji, i will correct it